procedures/katello/branch.md.erb

```diff
@@ -57,8 +57,9 @@
 - [ ] Push the KATELLO-<%= release %> branch using <%= rel_eng_script('branch_push') %>
 - [ ] Bump versions to <%= develop %>-master using <%= rel_eng_script('develop_branch_bump') %>
 - [ ] Push version bumps to <%= develop %>-master using <%= rel_eng_script('develop_branch_push') %>
-- [] Bump Katello related packages to their next versions
-  - [] [packaging](https://github.com/theforeman/foreman-packaging/pull/8729/files)
+- [ ] Bump Katello related packages to their next versions
+  - [ ] [packaging](https://github.com/theforeman/foreman-packaging/pull/8729/files)
+  - [ ] Update the katello-repos `pulpcore_version` to the latest branched version available in pulpcore-packaging: [Example from Katello 4.17](https://github.com/theforeman/foreman-packaging/blob/32fbf3e4287e36dcebcdc15add7cf067486d8228/packages/katello/katello-repos/katello-repos.spec#L1)
```

Comment:
@Odilhao do we also need to include the right GPG key?

Reply:
I don't recall anyone having to add GPG keys there

Reply:
I don't remember us proving the GPG there, the GPG for pulpcore was always imported using the http endpoint

https://github.com/theforeman/foreman-packaging/blob/rpm/3.15/packages/katello/katello-repos/katello.repo#L23